contents/handbook/people/offboarding.md

```diff
@@ -22,7 +22,20 @@ In this case, we are letting the team member go.
 
 This is generally for performance reasons or because the company's needs have changed and the role can no longer be justified. If the decision is down to performance issues, we will have already communicated feedback to the individual and given them time to take the feedback on board. However, performance issues sadly can't always be resolved, which means we might ultimately need to end someone's employment. 
 
-We will usually ask you to stop working immediately. If you have been with us for at least 3 months, you will be paid 4 months of salary as [severance](/handbook/people/compensation#severance). Otherwise, you will be paid 1 month of salary as severance. 
+Tim and James are responsible for making any final decision to let someone go. 
+
+We use the following general process for managing people whose performance isn't up to the right standard. We modify this slightly depending on the specific nature of the role and how long they have been at PostHog, so the process isn't identical in every case:
+
+- A member of the exec team will raise that a team member isn't meeting our performance expectations. They discuss what the issues are and a plan to improve. 
+- The relevant person on the exec team has a meeting with the team member to let them know explicitly that their performance is not meeting expectations, and and that if it continues then they will not be able to continue at PostHog. This meeting may include the person's manager, depending on role. 
+  - If the person is a manager, we usually collect feedback from their team beforehand. 
+- We outline to the team member exactly what good performance looks like (e.g. specific things to ship), and a timeline for improvement. Usually this is a few weeks for someone new to PostHog, but may be longer if the person has been at PostHog for a while. We schedule a time for a follow up conversation at the end of this period.
```

Comment:
I would probably expect them to come up with the plan after having received the feedback (esp in case of a lead), or certainly for it to be collaborative

Reply:
I've put collaborative - I think the level of collaboration in reality then depends on the seniority of the person. If more junior, I assume it'll be 'this is the plan, do you agree', whereas for a lead it's 'what is your plan to turn things around?'